### PR TITLE
Added AMQP Router to CircleCI

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Also added verbose flag to tests

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
Closes #236

**Comment by slinkydeveloper** [comment]
That's weird, locally pass (I tried even with `circleci local execute`), but it fails on CI for strange reasons...

**Comment by slinkydeveloper** [comment]
Uh ok now it passes :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,6 +3,25 @@ jobs:
   build:
     docker:
       - image: circleci/golang:1.13.4
+      - image: scholzj/qpid-dispatch
```
Comment (matzew) [inline .circleci/config.yml]: :+1:   good idea

**Comment by n3wscott** [comment]
SWEEET.

LGTM

**Pull request merged**